app/service/contact_svc.py

```diff
@@ -91,6 +97,12 @@ async def _save(self, result):
         except Exception as e:
             self.log.debug('save_results exception: %s' % e)
 
+    async def _postprocess_link_result(self, result, ability):
+        try:
+            return str(b64encode((await ability.HOOKS[ability.executor].postprocess(b64decode(result))).encode('utf-8')), 'utf-8')
```

Comment:
b64decode(result))).encode('utf-8')) is this not a util function? if not we do this a lot and may be good to create one